### PR TITLE
Update TS011F_plug_1.md

### PR DESCRIPTION
**Pull request by freeDom-**
Add information for updating to a custom firmware version as discussed in https://github.com/Koenkk/zigbee-OTA/pull/277

### CODE REVIEW

**Comment by freeDom-** [comment]
Recreated PR after https://github.com/Koenkk/zigbee2mqtt.io/pull/1950  was closed. Updated with comment on when to use as discussed in old PR, but found no way to reopen.

File: docs/devices/TS011F_plug_1.md [hunk 1]
```diff
@@ -66,6 +66,20 @@ Pair this device with a long press (5 seconds) on the on/off button. The button
 ## OTA updates
 This device supports OTA updates, for more information see [OTA updates](../guide/usage/ota_updates.md).
 
+An alternative firmware version can be installed by creating a custom my_index.json file and setting zigbee_ota_override_index_location as described in the [guide](../guide/usage/ota_updates.md#local-ota-index-and-firmware-files). This firmware can be used if the plug automatically turns off indeterminably or when the overvoltage protection from later versions is unwanted. To use v1.0.13 use the following json file:
```
Comment (Koenkk) [inline docs/devices/TS011F_plug_1.md:69]: Could you move this above "do not edit below"?

Comment (freeDom-) [reply docs/devices/TS011F_plug_1.md:65]: Added a new subchapter to notes to resolve this.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**